docs/content/en/docs/tasks/cluster/cluster-iam-auth.md

```diff
@@ -67,7 +66,7 @@ spec:
 ```
 
 {{% alert title="Note" color="primary" %}}
-When using backend mode `CRD`, the `mapRoles` and `mapUsers` are not required. For more details on configuring CRD mode, refer to [CRD](https://github.com/kubernetes-sigs/aws-iam-authenticator#crd-alpha).
+When using backend mode `CRD`, the `mapRoles` and `mapUsers` are not required. For more details on configuring CRD mode, refer to [CRD](https://github.com/kubernetes-sigs/aws-iam-authenticator#crd-alpha)
```

Comment:
I think it's okay to have a period at the end of that sentence. Is there a reason you removed it?

Reply:
Yea actually that was causing the alert box to get a new line i.e. there was a large blank section in the alert box. 
Which was kinda odd, and I was able to remove it by getting rid of the period.

Reply:
Okay.
/lgtm